packages/docs/examples/va-progress-bar/Slots.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="flex lg6 xs12">
+    <div class="mt-2 text--bold muted">Dynamic value:</div>
```

Comment:
Ideally we do not want to introduce any complicated texts in examples as they're no translated. So might be tricky for some users to understand.
Recommended approach - split into smaller examples or separate by prop values.

I.e.

```
## Sizes
`small` [ small example ]
`large` [ large example ]
```

is  all right, but

```
small component preset [ small example ]
large component preset [ large example ]
```

is not.